manon/collapsible.js

```diff
@@ -69,24 +69,35 @@ function makeResponsive(collapsibleElement, isCondensed) {
  * @return {{ element: HTMLButtonElement, setExpanded: (expanded: boolean) => void }}
  */
 function createMenuButton(collapsingElementId, openLabel, closeLabel, buttonClasses) {
-  var button = document.createElement("button");
-  button.className = "collapsible-toggle " + buttonClasses;
-  button.setAttribute("aria-controls", collapsingElementId);
-  button.setAttribute("aria-expanded", "false");
-  button.setAttribute("aria-haspopup", "menu");
+    var buttonElement = document.getElementById(collapsingElementId).parentElement.querySelector("button.collapsible-toggle");
+
+    console.log(buttonElement)
+
+    if(buttonElement) {
+      var button = buttonElement
+    } else {
+      var button = document.createElement("button");
+      button.className = "collapsible-toggle " + buttonClasses;
+    }
+
+    button.setAttribute("aria-controls", collapsingElementId);
+    button.setAttribute("aria-expanded", "false");
+    button.setAttribute("aria-haspopup", "menu");
 
-  var label = document.createElement("span");
-  label.innerText = openLabel;
-  label.className = "sr-only";
-  ensureElementHasId(label);
+    var label = document.createElement("span");
+    label.innerText = openLabel;
+    label.className = "visually-hidden";
+    ensureElementHasId(label);
 
-  button.appendChild(label);
-  button.setAttribute("aria-labelledby", label.id);
+    button.appendChild(label);
+    button.setAttribute("aria-labelledby", label.id);
 
   function setExpanded(expanded) {
     if (expanded !== (button.getAttribute("aria-expanded") === "true")) {
       button.setAttribute("aria-expanded", String(expanded));
-      label.innerText = expanded ? closeLabel : openLabel;
+      if(!buttonElement) {
+        label.innerText = expanded ? closeLabel : openLabel;
+      }
```

Comment:
This can be changed back to the original

```suggestion
      label.innerText = expanded ? closeLabel : openLabel;
```